packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3767,11 +3743,11 @@ export class ContainerRuntime
 	 * @param summaryLogger - logger to use when fetching snapshot from storage
 	 * @returns downloaded snapshot's reference sequence number
 	 */
-	private async refreshLatestSummaryAckFromServer(
```

Comment:
Find-all for `refreshLatestSummaryAckFromServer` also shows a comment in `runningSummarizer.ts`, can you take a look and update that to tell a more accurate story in light of this?

Reply:
Also the `scenarioName` param in the `getVersions` call should be updated to something else.  Maybe "prefetchLatestSummaryBeforeClose"?